### DISCUSSION TITLE
UX: simplify SEND flow

### DISCUSSION THREAD

**Discussion by tomzacchia** [cashubtc/eNuts#189] [Ideas]
**Is your feature request related to a problem? Please describe.**
Send > Send Ecash : shows a form to specify ecash amount to be sent

Send > Pay lightning invoice : navigates user to 3 actions 

**Requested Change**

Send > A single menu with menu groups, removes additional clicks to pay LN invoice and other lightning related send actions

Ecash
  - send Ecash
  
Lightning
  - LN invoice or LNURL
  - Scan QR Code
  - Mutimint

**Comment by KKA11010** [comment]
I've transformed this into a discussion. Thank you for the report, @tomzacchia.

Could you please clarify what you mean by 'a single menu with menu groups'? Could you provide some examples to illustrate your point?

It's important to consider that user actions are essential, and overcrowding a single screen with numerous options can potentially overwhelm the user.

**Comment by swedishfrenchpress** [comment]
Somewhat related to this: The current payment flow is designed so the 'payment overview' always pops up as the last step before completing a send. I really think the payment overview holds a lot of great info and we should totally keep it. But, having the user see and interact with it by default might not be the best UX. It's kind of an advanced screen and adds an extra tap to the send process.

Suggestion: Make the payment overview screen an optional step. Place it in the amount entry screen as a secondary action button. that slides up. This way, it's there if users want to see it, without adding an extra step for everyone else.
https://github.com/cashubtc/eNuts/assets/78821053/4de7ab05-565d-40a9-bb57-cbfda2543674
